zenpacklib.py

```diff
@@ -621,9 +624,21 @@ def _create_indexes(cls, zcatalog, spec):
                 # reindex all objects of this type so they are added to the
                 # catalog.
                 results = ICatalogTool(context).search(types=(classname,))
+
```

Comment:
There is actually another pull request open to do something similar (https://github.com/zenoss/zenpacklib/pull/150)  - Can you merge those other guards into this PR?